com.syntevo.SmartGit.json

```diff
@@ -0,0 +1,58 @@
+{
+  "app-id": "com.syntevo.SmartGit",
+  "command": "smartgit",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "separate-locales": false,
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--env=SMARTGIT_JAVA_HOME=/app/smartgit/jre",
+    "--filesystem=host",
+    "--share=ipc",
+    "--share=network",
+    "--socket=ssh-auth",
+    "--socket=wayland",
```

Comment:
Doubt this works on Wayland also.

Reply:
Sure XWayland is just X, this permission is about real Wayland. Yes remove it since it does nothing.